src/libs/PolicyUtils.ts

```diff
@@ -1232,6 +1232,10 @@ function getAllPoliciesLength() {
     return Object.keys(allPolicies ?? {}).length;
 }
 
+function getAllPolicies() {
+    return Object.values(allPolicies ?? {}).filter((p) => !!p);
```

Comment:
Most places we add a filter by calling `shouldShowPolicy`, is it needed here?

Reply:
I don't think so. This function should return all policies. The `filter` function is mainly to satisfy typescript.